src/sort.jl

```diff
@@ -0,0 +1,102 @@
+import Base.@_inline_meta
+import Base.Order: Ordering, Forward, ReverseOrdering, ord
+import Base.Sort: Algorithm, defalg, lt, sort
+
+
+struct BitonicSortAlg <: Algorithm end
+struct MinSizeSortAlg <: Algorithm end
+struct MinDepthSortAlg <: Algorithm end
+const MinSortAlg = Union{MinSizeSortAlg,MinDepthSortAlg}
+
+const BitonicSort = BitonicSortAlg()
+const MinSizeSort = MinSizeSortAlg()
+const MinDepthSort = MinDepthSortAlg()
+
+defalg(::SVector) = BitonicSort
+
+function sort(a::SVector;
+              alg::Algorithm = defalg(a),
+              lt = isless,
+              by = identity,
+              rev::Union{Bool,Nothing} = nothing,
+              order::Ordering = Forward)
+    ordr = ord(lt, by, rev, order)
+    println(Size(a))
+    (Size(a) == Size(0) || Size(a) == Size(1)) && return a
```

Comment:
Is the size wrapper necessary? Constant propagation/branch elimination should work for `length(a) == 0 || length(a) == 1 || return a`, especially on modern versions of Julia.